app/views/products/_form.html.haml

```diff
@@ -4,6 +4,8 @@
       %h3.card-title= form_header
       %hr
       = form_for([@shop, @product]) do |f|
+        = render 'shared/form_errors', f: @shop
```

Comment:
@danielpaul I'm sending the shop and products as f so that it will render errors based on the new products form. Otherwise it won't work